vendor_skins/Evercast/cmake/osxbundle/obslaunch.sh

```diff
@@ -1,12 +1,4 @@
 #!/bin/sh
-# use argument 1 as the version or get it from sw_vers
-os_ver=$(sw_vers -productVersion)
-
-if [[ "$os_ver" == 10.15.* ]]; then
-	echo "macOS Catalina"
-	osascript -e "tell app \"Terminal\" to do script \"cd /Applications/EBS.app/Contents/Resources/bin && ./ebs\""
-else
-	cd "$(dirname "$0")"
-  cd ../Resources/bin
-  exec ./ebs "$@"
-fi
+cd "$(dirname "$0")"
+cd ../Resources/bin
```

Comment:
Hm.. are we still using obslaunch.sh? I was under the impression this was not being bundled anymore. Maybe it's not?

Reply:
You're right, it's not being bundled.  I'll remove the vendor_skins version.